sdk/rust/build.rs

```diff
@@ -8,17 +8,14 @@ fn main() {
         format!("-{pre}")
     };
 
-    let output = Command::new("git")
+    let commit: String = Command::new("git")
```

Comment:
Nit: is there any reason for the `: String` annotation?

Reply:
No. Thanks for pointing it out. it was remnant from me playing around with the mapping.